lib/rubocop/cop/rails/migration_file_name.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop makes sure that each migration file defines a migration class
+      # whose name matches the file name.
+      # (i.e. `20220224111111_create_users.rb` should define `CreateUsers` class.)
+      #
+      # @example
+      #   # db/migrate/20220224111111_create_users.rb
+      #
+      #   # bad
+      #
+      #   class SellBooks < ActiveRecord::Migration[7.0]
+      #   end
+      #
+      #   # good
+      #
+      #   class CreateUsers < ActiveRecord::Migration[7.0]
+      #   end
+      #
+      class MigrationFileName < Base
```

Comment:
`MigrationClassName` seems to be more suitable than `MigrationFileName`.

```suggestion
      class MigrationClassName < Base
```

Reply:
That's absolutely right! Thanks!

Reply:
I edited the title of this pull request as well.